app/src/main/java/com/markkolenbrander/capstonenewsapp/composable/ArticleListComposable.kt

```diff
@@ -0,0 +1,155 @@
+package com.markkolenbrander.capstonenewsapp.composable
+
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Divider
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.google.accompanist.swiperefresh.SwipeRefresh
+import com.google.accompanist.swiperefresh.rememberSwipeRefreshState
+import com.markkolenbrander.capstonenewsapp.R
+import com.markkolenbrander.capstonenewsapp.composetheme.AppTheme
+import com.markkolenbrander.capstonenewsapp.models.*
+import com.skydoves.landscapist.ImageOptions
+import com.skydoves.landscapist.glide.GlideImage
+import kotlinx.coroutines.delay
+
+
+@Composable
+fun ArticleNewsItem(article: List<Article>, onItemClicked: (Article) -> Unit) {
+
+    var refreshing by remember { mutableStateOf(false) }
+    LaunchedEffect(refreshing) {
+        if (refreshing) {
+            delay(1000)
+            refreshing = false
+        }
+    }
+
+    SwipeRefresh(
+        state = rememberSwipeRefreshState(isRefreshing = refreshing),
```

Comment:
Just a personal preference, but I typically like to set the "remembered" objects as variables and move those to the top of the composable function.  So this would be `val swipeRefreshState by rememberSwipeRefreshState(isRefreshing = refreshing)` and then here:

```suggestion
        state = swipeRefreshState,
```